clase6/ej1.js

```diff
@@ -0,0 +1,80 @@
+const $botonMiembros = document.querySelector("#botonMiembros");
+const $formulario = document.querySelector("#formulario");
+const $cantMiembros = document.querySelector("#cantMiembros");
+const $edadMayor = document.querySelector("#edadMayor");
+const $edadMenor = document.querySelector("#edadMenor");
+const $edadPromedio = document.querySelector("#edadPromedio");
+const $botonCalcular = document.querySelector("#botonCalcular");
+const $salarioMayor = document.querySelector("#salarioMayor");
+const $salarioMenor = document.querySelector("#salarioMenor");
+const $salarioPromedio = document.querySelector("#salarioPromedio");
+
+$botonMiembros.onclick = function () {
+  const cantMiembros = Number($cantMiembros.value);
+
+  for (let i = 0; i < cantMiembros; i++) {
+    const $nuevaLabelEdad = document.createElement("label");
+    $nuevaLabelEdad.textContent = `Ingrese edad del ${i + 1}° integrante: `;
+
+    const $nuevoInputEdad = document.createElement("input");
+    $nuevoInputEdad.type = "number";
+    $nuevoInputEdad.id = "edades";
```

Comment:
Los ids **deberían ser únicos**, no deberíamos compartir ids entre elementos html, lleva a comportamientos inesperados, para eso podemos usar `class`. Te dejo unos links para que chusmees:
https://stackoverflow.com/questions/9233191/repeated-ids-in-an-html-document-how-bad-an-idea-is-it-if-they-are-scoped-by
https://softwareengineering.stackexchange.com/questions/127178/two-html-elements-with-same-id-attribute-how-bad-is-it-really